src/views/Logbook/LogbookDetail/LogbookEditor/styles.module.css

```diff
@@ -16,9 +16,13 @@
 
   & img {
     display: block;
-    width: 351px;
+    width: 251px;
     height: auto;
     margin: 0 auto;
+
+    @media (--md-h-up) {
+      width: 351px;
```

Comment:
I see the intro img is using `200px` and `288px` but editor's img `351px` is too big in `wide and low` viewport cases, should we align to use the same ?